Exercises/arrays-methods (from gist)/arrays-methods (from gist).js

```diff
@@ -0,0 +1,100 @@
+//filter
+function allNonConsecutive (arr) {
+    return arr.filter((item, index) => index && arr[index - 1] !== item - 1).map((item, index) => ({i: arr.indexOf(item), n:item}));
+}
+
+//every
+const isValid = (array, validator) => {
+    return array.every(validator);
+    }
+
+//some
+// I dont know how to solve this task using some ;(
+const filterBlockedProducts = (products, blockedBrands) => {
+    return products.filter((item) => !blockedBrands.includes(item.brand))
+}
```

Comment:
You can use some instead of includes with some changes to check if there some blockedBrands which are same as current checked product name.